test/module/shared_model/backend_proto/shared_proto_tx_response_test.cpp

```diff
@@ -46,3 +48,32 @@ TEST(ProtoTxResponse, TxResponseLoad) {
               shared_model::crypto::Hash(hash));
   });
 }
+
+/**
+ * @given TransactionResponse that previously had lazy fields
+ * @when those lazy fields are simultaneously accessed
+ * @then there is no race condition and segfaults
+ */
+TEST(TxResponse, SafeToReadFromMultipleThreads) {
+  iroha::protocol::ToriiResponse response;
+  const std::string hash = "123";
```

Comment:
Seems that var used only once so it might be inlined

Reply:
You are right. It could be inlined. I would like leave it as is, since it does not violate any of our guides.